src/gr11review/part1/Review1.java

```diff
@@ -1 +1,61 @@
 package gr11review.part1;
+
+import java.io.*;
+
+public class Review1{
+
+    public static void main(String[] args) throws IOException{
+        BufferedReader month = new BufferedReader(new InputStreamReader(System.in));
+        BufferedReader day = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
You are creating two BufferedReader objects, month and day, but the test is setting a single InputStream. This results in the second BufferedReader not being properly initialized with the input provided by the test, causing the NumberFormatException.

You only need one BufferedReader for reading both the month and day inputs, as both will come from the same input stream (System.in). For example:

```
BufferedReader reader = new BufferedReader(new InputStreamReader(System.in)); // Single reader
```

Then reference the same `reader` object for both month and day.